packages/dkml-component-staging-opam32/dkml-component-staging-opam32.2.2.0~dkml20220801/opam

```diff
@@ -288,9 +288,11 @@ extra-source "dl/opam/src_ext/archives/sha-1.15.2.tbz" {
 # END OPAM ARCHIVES
 # -------------------
 extra-source "dl/dkml-runtime-common.tar.gz" {
-  src: "https://github.com/diskuv/dkml-runtime-common/archive/refs/tags/v1.0.0.tar.gz"
+  # This violates the v1.0.0 versioning, but this future patch is needed to resolve build
+  # issues on macOS in Opam CI.
```

Comment:
This is not just an opam issue. Regular users will encounter this issue. My guess for why you didn't encounter that yourself is that you ran `brew install` in the past couple of days before testing.
```suggestion
  # issues on macOS in Opam.
```